Cargo.toml

```diff
@@ -16,23 +16,9 @@
 # under the License.
 
 [workspace]
-members = [
-    "datafusion",
-    "datafusion-common",
-    "datafusion-expr",
-    "datafusion-jit",
-    "datafusion-physical-expr",
-    "datafusion-cli",
-    "datafusion-examples",
-    "datafusion-proto",
-    "benchmarks",
-    "ballista/rust/client",
-    "ballista/rust/core",
-    "ballista/rust/executor",
-    "ballista/rust/scheduler",
-    "ballista-examples",
+members = ["datafusion", "datafusion-common", "datafusion-expr", "datafusion-jit", "datafusion-physical-expr", "datafusion-cli", "datafusion-examples", "datafusion-proto", "benchmarks", "ballista/rust/client", "ballista/rust/core", "ballista/rust/executor", "ballista/rust/scheduler", "ballista-examples",
```

Comment:
is there a way to configure the fmt check so that it allows multi-line lists? moving all entries into a single line increases the chances of conflicts and makes it harder to read too.